R/lint.R

```diff
@@ -737,7 +733,7 @@ highlight_string <- function(message, column_number = NULL, ranges = NULL) {
   line <- fill_with(" ", maximum)
 
   lapply(ranges, function(range) {
-    substr(line, range[1L], range[2L]) <<-
+    substr(line, range[1L], range[2L]) <<- # nolint: undesirable_operator_linter.
       fill_with("~", range[2L] - range[1L] + 1L)
```

Comment:
Is the indentation lint here a false positive? AFAICT, the indentation should be 6 spaces here.

> Indentation should be 4 spaces but is 6 spaces.

Btw, if I try to inline this, I create `line_length_linter()` lint.

Reply:
cc @AshesITR

Reply:
It is, seems like the comment throws off the XPath for indentation changes.

Reply:
Thanks for confirming. I will wait for the related issue to be resolved. Then we can revisit this PR.